namespace cpp entity
namespace py ccentity.contacts
namespace as3 com.dreamfire.sanguo.data.definition.entity

////////////////////////////////////////////////
// 联系人类型定义
enum ContactsType {
  MIN = 0,

  FRIEND = 0,                         // 好友
  BLACKLIST,                          // 黑名单

  MAX,
}

